### PR TITLE
Admin Generator (Future): Support `helperText` `Field` prop in Forms

### PR DESCRIPTION
**Pull request by andrearutrecht**
Support `helperText` `Field` prop in Forms
<details><summary>Screenshot of Demo:</summary>
<p>

Screenshot 2024-02-15 at 09 06 28

</p>
</details> 

- [ ] Add changeset (if necessary)

### CODE REVIEW

**Review by Ben-Ho — APPROVED** [review]

**Comment by nsams** [comment]
do you want review for this? (becasue it's draft)

and please target next

**Comment by andrearutrecht** [comment]
I branched away from `admin-gen-configurable`. Should this really be targeting `next` right now?

**Comment by nsams** [comment]
> I branched away from admin-gen-configurable. Should this really be targeting next right now?

yes, admin-gen-configurable was merged into next (and even if it wouldn't you must target next)

and it was squashed, you need to rebase correctly

**Comment by andrearutrecht** [comment]
Is there a preference for `merge` or `rebase` for Comet?

**Comment by nsams** [comment]
in this case you have no option than rebasing, as the base branch was squash-merged

**Comment by andrearutrecht** [comment]
Rebasing is done

**Comment by nsams** [comment]
pipeline is failing.

And you also  have to verify your changes by running and testing in demo

**Review by nsams — APPROVED** [review]

**Review by johnnyomair — APPROVED** [review]

**Comment by johnnyomair** [comment]
@andrearutrecht please merge next to resolve conflicts, thanks!

**Comment by andrearutrecht** [comment]
@Ben-Ho @johnnyomair  @nsams Conflicts are resolved 👍

File: demo/admin/src/products/future/ProductForm.cometGen.ts [hunk 1]
```diff
@@ -13,11 +13,13 @@ export const ProductForm: FormConfig<GQLProduct> = {
             required: true, // default is inferred from gql schema
             validate: { name: "validateTitle", import: "./validateTitle" },
         },
+        // "packageDimensions.height" naming is not yet supported.
+        // { type: "text", name: "packageDimensions.height", label: "Height", helperText: "Enter height in centimeters" },
```
Comment (nsams) [inline demo/admin/src/products/future/ProductForm.cometGen.ts:17]:
```suggestion
```

**Review by nsams — APPROVED** [review]

**Review by johnnyomair — APPROVED** [review]

**Pull request merged**